superset-frontend/src/visualizations/presets/MainPreset.js

```diff
@@ -88,10 +87,10 @@ export default class MainPreset extends Preset {
         new StatusIndicatorChartPlugin().configure({key: 'status_indicator'}),
         new IframeDemoChartPlugin().configure({key: 'iframe_demo'}),
         new CccsGridChartPlugin().configure({key: 'cccs_grid'}),
+        new BigNumberChartPlugin().configure({key: 'big_number'}),
+        new BigNumberCurrencySelectionChartPlugin().configure({key: 'big_number_total'}),
         new AreaChartPlugin().configure({ key: 'area' }),
         new BarChartPlugin().configure({ key: 'bar' }),
-        new BigNumberChartPlugin().configure({ key: 'big_number' }),
-        new BigNumberTotalChartPlugin().configure({ key: 'big_number_total' }),
         new EchartsBoxPlotChartPlugin().configure({ key: 'box_plot' }),
```

Comment:
we might want to have both of the charts registered. Like out custom big number and their legacy big number